tests/viz/test_exporter.py

```diff
@@ -14,43 +14,48 @@
 
 import os
 import shutil
-import xml.etree.ElementTree as ET
-from collections import namedtuple
+from dataclasses import dataclass
 from pathlib import Path
-from typing import Union
 
-import meshio
 import numpy as np
 import pytest
-from deepdiff import DeepDiff
 
 import porepy as pp
+from porepy.applications.test_utils.vtk import (
+    PathLike,
+    compare_pvd_files,
+    compare_vtu_files,
+)
 from porepy.fracs.utils import pts_edges_to_linefractures
 
```

Comment:
Perhaps a matter of taste, but I prefer `folder_reference` instead of capitalizing variable names.